app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -1,247 +1,191 @@
 //= depend_on_asset "active_admin/datepicker/datepicker-input-icon.png"
-// -------------------------------------- Active Admin Forms
-form {
-  /* Reset margins & Padding */
-  ul, ol, li, fieldset, legend, input, textarea, select, p { margin:0; padding:0; }
-  ol, ul { list-style: none }
+
+input[type=text],
+input[type=password],
+input[type=email],
+input[type=number],
+input[type=url],
+input[type=tel],
+textarea {
+  border: $border-width solid #c9d0d6;
+  border-radius: 3px;
+  display: inline-block;
+  outline: none;
+  padding: 6px 8px;
+  width: calc(80% - #{$text-input-total-padding});
+
+  &:focus {
+    border: $border-width solid #99a2aa;
+    box-shadow: 0 0 4px #99a2aa;
+  }
+}
+
+.formtastic {
+  ol, ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .input {
+    margin-bottom: 15px;
+  }
+
+  .label {
+    color: $form-label-color;
+    display: block;
+    float: left;
+    font-weight: bold;
+    width: 20%;
+
+    abbr {
+      border: none;
+      color:  $required-field-marker-color;
+      margin-left: 0.3em;
+    }
+  }
+
+  .inputs {
+    @include section-background;
+
+    > ol {
+      padding: 15px;
+    }
+  }
 
   fieldset {
     border: 0;
-    padding: 10px 0;
-    margin-bottom: 20px;
-
-    &.inputs { @include section-background; }
+    margin: 0 0 20px 0;
+    padding: 0;
+    position: relative; // for abosolute positioning sub fieldset legends for choices/fragments
 
     legend {
       width: 100%;
-      span { display: block; @include section-header; }
-    }
 
-    ol > li {
-      padding: 10px;
-      label {
+      span {
         display: block;
-        width: 20%;
-        float: left;
-        font-size: 1.0em;
-        font-weight: bold;
-        color: $form-label-color;
-        abbr {
-          border: none;
-          color:  $required-field-marker-color;
-        }
+        @include section-header;
       }
     }
 
-    ol > li.has_many_container {
-      padding: 20px 10px;
+    .has_many_container {
       h3 {
-        font-size: 12px;
-        font-weight: bold;
-      }
-      .has_many_fields { margin: 10px 0 }
-    }
-
-    ol > li > li label {
-      line-height:100%;
-      padding-top:0;
-      input {
-        line-height:100%;
-        vertical-align:middle;
-        margin-top:-0.1em;
+        border-bottom: 1px solid #e8e8e8;
```

Comment:
Color literals like `#e8e8e8` should only be used in variable declarations; they should be referred to via variable everywhere else.